opentech/users/groups.py

```diff
@@ -0,0 +1,35 @@
+GROUPS = [
+    {
+        'name': 'Applicant',
+        'permissions': [],
+    },
+    {
+        'name': 'Reviewer',
+        'permissions': [],
+    },
+    {
+        'name': 'Advisor',
+        'permissions': [],
+    },
+    {
+        'name': 'Staff',
+        'permissions': [],
+    },
+    {
+        'name': 'Manager',
+        'permissions': [
+            'add_image', 'change_image', 'delete_image',
+            'add_document', 'change_document', 'delete_document',
+            'access_admin'
```

Comment:
Can you split this so that there is one permission per line. make life easier in the long run.